addons/custom-menu-bar/addon.json

```
@@ -0,0 +1,133 @@
+{
+  "name": "Custom menu bar",
```

Most addons use customizable
```suggestion
  "name": "Customizable menu bar",
```